libs/wire-api/src/Wire/API/User.hs

```diff
@@ -701,11 +701,10 @@ data BindingNewTeamUser = BindingNewTeamUser
   deriving (Arbitrary) via (GenericUniform BindingNewTeamUser)
 
 instance ToJSON BindingNewTeamUser where
-  toJSON (BindingNewTeamUser t c) =
-    let (Object t') = toJSON t
-     in object $
-          "currency" .= c
-            # HashMap.toList t'
+  toJSON (BindingNewTeamUser (BindingNewTeam t) c) =
+    object $
+      "currency" .= c
+        # newTeamJson t
```

Comment:
huh?  why has the indentation changed here?

Reply:
Not just the indentation, but the code changed. :stuck_out_tongue: 

I got rid of the `let`.